examples/textinput/textinput/app.py

```diff
@@ -29,11 +30,13 @@ def do_extract_values(self, widget, **kwargs):
             self.password_input.value,
         )
 
-        number = self.number_input.value
-        if number:
+        number = self.number_input.value + self.right_aligned_number_input.value
+        if number > 0:
             self.number_label.text = f"Double the number is: {number * 2}"
         else:
-            self.number_label.text = "You didn't enter a number"
+            self.number_label.text = (
+                "Pick some other numbers that won't add up to zero."
```

Comment:
This is a misleading message, because that's not what you're testing with the if.

Reply:
How about "Pick some numbers that add to more than zero"?

Reply:
I've updated the label to show Pick some numbers that add up to more than zero